R/install_easystats.R

```diff
@@ -51,8 +53,11 @@ install_latest <- function(source = c("development", "cran"),
 
   if (source == "development") {
     repos <- "https://easystats.r-universe.dev"
-  } else {
+  } else if (source == "cran") {
     repos <- getOption("repos")["CRAN"]
+  } else if (insight::check_if_installed("pak")) {
+    pak::pkg_install(paste0("easystats/", packages), dependencies = FALSE)
```

Comment:
This would skip the version printing that the two other methods have. Any chance this can done later in the function so that version numbers of packages to update are still printed?

Also, if `pak` is not installed then this branch is ignored and the rest of the function runs. We should error if `source = "github"` and `pak` is not installed.

Reply:
Agree with the printing, that also came to my mind.
But if pak is not installed, `check_if_installed()` errors.